src/components/navigation/navigation-list/navigation-list.ts

```diff
@@ -52,18 +39,8 @@ export class SbbNavigationListElement extends SlotChildObserver(LitElement) {
     );
   }
 
-  public override connectedCallback(): void {
-    super.connectedCallback();
-    this._handlerRepository.connect();
-  }
-
-  public override disconnectedCallback(): void {
-    super.disconnectedCallback();
-    this._handlerRepository.disconnect();
-  }
-
   protected override render(): TemplateResult {
-    const hasLabel = !!this.label || this._namedSlots['label'];
+    const hasLabel = !!this.label || this._namedSlots.slots.has('label');
```

Comment:
maybe we don't need this condition anymore, as the id is now alwys there and the reference always valid. if it's empty, it's up to the screen reader to read nothing I think. maybe test it?